schema/mysql/v57/temporal/schema.sql

```diff
@@ -238,6 +238,8 @@ CREATE TABLE cluster_metadata (
   metadata_partition        INT NOT NULL,
   immutable_data            BLOB NOT NULL,
   immutable_data_encoding   VARCHAR(16) NOT NULL,
+  mutable_data              BLOB,
+  mutable_data_encoding     VARCHAR(16),
```

Comment:
mutable_data BLOB NOT NULL,
mutable_data_encoding VARCHAR(16) NOT NULL,

Reply:
Existing code should be able to write to new database. Making it `NOT NULL` will fail the write operation there.

Reply:
> Existing code should be able to write to new database. Making it `NOT NULL` will fail the write operation there.

Not really, existing running code should not touch the db since the initialization is already done. During deployment, if the new code does read then write (optionally), there will be no issue. 

I am fine with this PR btw.

Reply:
I've refactored whole implementation, which allowed us to make this field 'NOT NULL`, please take a look